specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/examples/QueryPackQueriesSearch.json

```diff
@@ -0,0 +1,65 @@
+{
+  "parameters": {
+    "api-version": "2019-09-01-preview",
+    "subscriptionId": "86dc51d3-92ed-4d7e-947a-775ea79b4918",
+    "resourceGroupName": "my-resource-group",
+    "queryPackName": "my-querypack",
+    "includeContent": true,
```

Comment:
>"includeContent": true, [](start = 4, length = 23)

what additional content is included or excluded by this parameter?

Reply:
I changed this to **includeBody**. 
According to our experience the body of the query can get quite large, so to decrease the size of the payload for efficiency we allow this parameter. In addition, in some flow for UI purposes all that is needed is metadata of the query and not the body itself.